FiftyOne.Pipeline.Core/FlowElements/PipelineBuilder.cs

```diff
@@ -402,7 +410,7 @@ private void AddElementToList(
         /// The index of the element within the <see cref="PipelineOptions"/>.
         /// </param>
         private void AddParallelElementsToList(
```

Comment:
This is cosmetics, but I would rename this method into `EnqueParallelElements` since now it is not adding to the list but adding to the queue